driver/main_args.mli

```diff
@@ -254,3 +254,11 @@ module Make_bytetop_options (F : Bytetop_options) : Arg_list;;
 module Make_optcomp_options (F : Optcomp_options) : Arg_list;;
 module Make_opttop_options (F : Opttop_options) : Arg_list;;
 module Make_ocamldoc_options (F : Ocamldoc_options) : Arg_list;;
+
+(** [options_with_command_line_syntax options r] returns [options2] that behaves
+    like [options], but additionally pushes command line argument on [r] (quoted
+    by [Filename.quote] when necessary). *)
```

Comment:
Maybe we could add a comment that this is used by ocaml{c,opt}p for echoing arguments to ocaml{c,opt}?

Reply:
Sure.